src/apps/SafeConnect/components/WCClientInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components";
+import { Text, IconText, Icon } from "@gnosis.pm/safe-react-components";
+
+const FlexAlignedCenter = styled.div`
+  display: flex;
+  margin-bottom: 10px;
+
+  > :first-of-type {
+    margin-right: 5px;
+  }
+`;
```

Comment:
there are no alignment styles but the name says so

Reply:
yes, I removed the alignment and I forget to rename the component.